docs/page-objects/items.rst

```diff
@@ -42,6 +42,12 @@ To keep your code maintainable, we recommend you to:
     :ref:`page object classes <page-object-classes>`, e.g. using :ref:`field
     processors <field-processors>`.
 
+    Having code that makes item field values different from their input values
+    could hinder the use of future web-poet features, such as `field subset
+    selection`_.
```

Comment:
I was thinking mostly about current features: if you have a page object, and `value = await page.field` is not the same as `item = await page.to_item(); value = item.field`, then it could lead to issues in practice, when people are expecting a different behavior. Accessing `page.field` is the current way to do the field subset selection.

Reply:
https://github.com/scrapinghub/web-poet/pull/185/commits/421e83874c490ac8baf9be1c0673ab9255f1ff9b ?